src/resources/js/flexible-tickets/modules/filters.js

```diff
@@ -216,3 +216,28 @@ wp.hooks.addFilter (
     'tec.tickets.flexibleTickets',
     filterTicketsControlsMappedProps
 )
+
+/**
+ *
+ * Forces the Prompt Message component to show on Recurring Events if they are part of a Series.
+ *
+ * @since TBD
+ *
+ * @param {ReactNode} component
+ * @returns
+ */
+function filterDisplaySeriesPassesPromptMessage( component ) {
+	const hasSeriesPasses = ( TECFtEditorData?.series?.seriesPassesCount || 0 ) > 0;
+
+	if ( hasSeriesPasses ) {
+		return component;
+	}
+
+	return null;
```

Comment:
Else replace the mapped properties with `null`? How does this work in Events that are not part of Series or posts that are not Events?

Reply:
This file was deleted, nothing to do here